app/frontend/stores/jurisdiction-store.ts

```diff
@@ -84,18 +84,39 @@ export const JurisdictionStoreModel = types
         self.environment.api.searchJurisdictions({
           query: self.query,
           sort: self.sort,
-          page: opts?.page ?? self.currentPage,
-          perPage: opts?.countPerPage ?? self.countPerPage,
           filters: {
             submissionInboxSetUp,
           },
         })
       )
       if (response.ok) {
-        self.mergeUpdateAll(response.data.data, "jurisdictionMap")
-        self.tableJurisdictions = cast(response.data.data.map((jurisdiction) => jurisdiction.id))
-        self.currentPage = opts?.page ?? self.currentPage
-        self.countPerPage = opts?.countPerPage ?? self.countPerPage
+        const camelizedData = keysToCamelCase(response.data.data) as IJurisdiction[]
```

Comment:
Whenever AI suggests that our uncamelized backend keys are causing a problem, you need to explain to it that our keys are already being transformed. Is that what's happening here?